app/src/main/java/com/edu/uni/augsburg/uniatron/stepcounter/StepCountService.java

```diff
@@ -0,0 +1,97 @@
+package com.edu.uni.augsburg.uniatron.stepcounter;
+
+import android.app.Service;
+import android.content.Intent;
+import android.hardware.Sensor;
+import android.hardware.SensorEvent;
+import android.hardware.SensorEventListener;
+import android.hardware.SensorManager;
+import android.os.AsyncTask;
+import android.os.IBinder;
+import android.support.annotation.Nullable;
+
+//import com.edu.uni.augsburg.uniatron.MainApplication;
+
+/**
+ * The step count service collects steps and commits them to the database.
+ *
+ * @author Leon Wöhrl
+ */
+public class StepCountService extends Service implements SensorEventListener {
+
+    private SensorManager sensorManager;
+    private Sensor stepDetectorSensor;
+    private static final int COMMIT_SIZE = 10;
```

Comment:
For a better understanding of the class, move all constants up as first variables.